pages/ugm2025.md

```diff
@@ -0,0 +1,168 @@
+
+## Keynote Speakers
+
+Keynote speakers will be announced soon.
+
+## Program
+
+A finalized agenda will be available closer to the event date. The UGM will include a mix of talks, tutorials, and a hackathon.
+
+## Travel and Accommodation
+
+*Participants are expected to make their own travel and accommodation arrangements.*
+
+[Tempe][] is part of the "Valley of the Sun" metropolitan area, which encompasses
+[Phoenix][], the 5th most populous city in the US. Today the area is home to more
+than 4.8 million people and encompasses the ancient lands of the Akimel O'odham
+(Pima), Maricopa, and Tohono O'odham tribes.
+
+### Airport
+
+**Phoenix Sky Harbor** [PHX][] airport has direct connections to all major hubs
+in the US.
+
+PHX airport is 5.5 miles (10 minutes by car/rideshare) from the workshop venue
+
+### Public transport (Light Rail)
+
+The [Valley Metro Light Rail][Light Rail] connects the airport to Tempe along a
+West-East axis. If your destination is near a Light Rail stop then
+public transport works well:
+
+1. Take the free [PHX Sky Train][] from the terminal to the **44th St/Washington**
+Metro Rail stop.
+2. Walk from the Sky Train stop to the Light Rail stop (all inside the 44th St
+Sky Train structure)
+3. Buy a ticket at the ticket machine near the exit escalators to the Light
+Rail stop.
+4. At the [44th St/Washington stop][LightRail44thSt] (Stop #10018) take the
+*east-bound* Light Rail (direction Mesa, Gilbert Rd/Main St).
+5. The stop closest to the Hyatt Place Hotel and the workshop venue is
+**University Dr/Rural Rd** (Stop #10023). If you want to get off closer to
+downtown Tempe get off one stop earlier at **Veterans Way/College Ave** (Stop #10025)
+6. Walk to your destination.
+
+### Car/rideshare
+
+To get from the airport to anywhere in Tempe either take a cab (traditional or self-driving
+Waymo) or rideshare (Lyft, Uber).
+
+### ASU Venue
+
+The workshop will be held in the Student Success Center in the Department of
+Physics at [Arizona State University][ASU] (room PSF 186 in the [Physical Sciences F
+building][PSF]). The address is
+
+  Department of Physics  
+  Arizona State University  
+  Physical Sciences F Building  
+  550 E Tyler Mall, PSF 186  
+  Tempe, AZ 85287  
+  USA
+ 
+Paid parking is available in the [Tyler Street Parking
+Structure][TylerStParking] and other on-campus parking structures.
+
+## Support for international travel
+Attendees requiring a letter clarifying the purpose of travel should [contact the MDAnalysis project manager][email]. 
+
+## Meals
+More information on local food options will be added soon!
+
+## Health and Safety
+We are committed to providing a safe, comfortable, and healthy environment for the MDAnalysis UGM and have looked to local and international entities and other organizations in the open source community to develop onsite safety measures for UGM attendees.
+
+### COVID-19 Testing
```

Comment:
I think we can get rid of this?

Reply:
I think we normally keep it in for people as standard policy at our events